vite.config.ts

```diff
@@ -12,12 +18,29 @@ export default defineConfig({
     },
   },
   plugins: [
-    react(), 
-    svgr(),
+    react(),
+    svgr({
+      include: '**/*.svg',
+      svgrOptions: {
+        exportType: 'default',
+      },
+    }),
     dts(),
+    visualizer({
+      filename: './dist/report.html',
+      brotliSize: true,
+    }),
+    // pluginPurgeCss({
+    //   variables: true,
+    // }),
```

Comment:
lets remove unused

Reply:
I commented out this since we're talking about the purgecss in here https://sendbird.slack.com/archives/C05QD5M7XDH/p1714457280742949?thread_ts=1714383967.760279&cid=C05QD5M7XDH. TBD yet 🤔

Reply:
Removed it https://github.com/sendbird/chat-ai-widget/pull/180/commits/26d1e7283ae06399b12a079cd4ce2e8d2bb02f13 :)